src/components/list-item/list-item.tsx

```diff
@@ -99,21 +199,35 @@ export class ListItem implements ConditionalSlotComponent, InteractiveComponent
   //
   // --------------------------------------------------------------------------
 
+  renderExpand(): VNode {
+    const { el, expanded, expandable } = this;
+    const dir = getElementDir(el);
+
+    return expandable ? (
+      <td class={CSS.expandContainer} onClick={this.toggleExpanded}>
+        <calcite-icon
+          icon={expanded ? "caret-down" : dir === "rtl" ? "caret-left" : "caret-right"}
+          scale="s"
+        />
+      </td>
+    ) : null;
+  }
+
   renderActionsStart(): VNode {
     const { el } = this;
     return getSlotted(el, SLOTS.actionsStart) ? (
-      <div class={CSS.actionsStart}>
+      <td class={CSS.actionsStart} ref={(el) => (this.actionsStartEl = el)} role="gridcell">
         <slot name={SLOTS.actionsStart} />
```

Comment:
Coupled with adding an `aria-label` to the slot:
`<slot aria-label={label} name={SLOTS.actionsStart} />`